packages/reports/tests/integration/components/column-selector-test.ts

```diff
@@ -0,0 +1,135 @@
+import { click, fillIn, findAll, render, triggerEvent } from '@ember/test-helpers';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupRenderingTest } from 'ember-qunit';
+import { TestContext as Context } from 'ember-test-helpers';
+import { Server } from 'miragejs';
+import hbs from 'htmlbars-inline-precompile';
+import FragmentFactory from 'navi-core/services/fragment-factory';
+import ColumnMetadataModel from 'navi-data/models/metadata/column';
+import NaviMetadataService from 'navi-data/services/navi-metadata';
+import { module, test } from 'qunit';
+import config from 'ember-get-config';
+//@ts-ignore
+import { assertTooltipRendered, assertTooltipNotRendered, assertTooltipContent } from 'ember-tooltips/test-support';
+
+interface TestContext extends Context {
```

Comment:
Maybe use this pattern for including component args
https://github.com/yavin-dev/framework/blob/c3748bf1d035864954b1a3bb7709b12646a87732/packages/reports/tests/integration/components/report-view-overlay-test.ts#L6-L10